frontend/src/filter-menu.tsx

```diff
@@ -53,7 +53,7 @@ export function FilterMenu({urlParam, buttonText, extractValueFromPull}: FilterM
              <MenuItemOption
                 key={value}
                 value={value}>
-                {value} ({valueToPullCount.get(value) || 0})
+                {value} ({valueToPullCount[value] || 0})
```

Comment:
I think it would be a little nicer to leave the count off here and show this item grayed out. I see little benefit in clicking a filter we know will yield zero results (at least, at present), so I think it makes sense to remove some visual weight relative to other items.

Reply:
When building a filter, you may want to include authors that don't *currently* have any pulls but you'd like to include them. Our team uses a link that includes `author=.....` a big list. Not all of them always have pulls open. FYI, 0s will pretty much only be present if there are authors included in the url that don't have open pulls.